app/utility/enum_mixin.py

```diff
@@ -0,0 +1,10 @@
+import enum
+
+
+class IterableEnumMixin(
```

Comment:
`IndexableStringEnum` may be?

Reply:
JobStatus is modified too, but in actually code we hadn't used comparing job status enum for now